portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,19 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let titles = [ 'Freelance Web Developer', 'Robotics Builder', 'Graphic Designer', 'Entrepreneur' ];
+let i = 0;
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+setInterval(changeTitle, 3000);
+
+
+function  changeTitle(){
```

Comment:
Do you have a description for this method anywhere? I can infer what it's trying to do, but the method names aren't that obvious.

Reply:
I did not put a comment to describe the name of the function as I thought it was a good name.

The function is supposed to go through each title that describes me every 3 seconds. What would a better name be for this?

Reply:
I don't there is a problem with the name (although it could be less vague). Based on your description, and data structure on line 19, it seems fine. One possibly way to make these less vague is: s/titles/previousPositions and then this becomes changePositionDisplayed etc.

Reply:
that makes sense I will fix that thank you, for the insight